packages/common/test/hardforks.spec.ts

```diff
@@ -305,10 +305,10 @@ describe('[Common]: Hardfork logic', () => {
         terminalTotalDifficulty: 0,
         shanghaiTime: 0,
         cancunTime: 0,
-      },
-      difficulty: '100',
+      } as GethGenesisConfig,
+      difficulty: '0x100',
       alloc: {},
-      gasLimit: '5000',
+      gasLimit: '0x5000',
```

Comment:
Nit: 5000 != 0x5000, value change might not be part of test though so it might be fine

Reply:
same as other, it's unused